src/store/index.js

```diff
@@ -115,5 +115,12 @@ export default new Vuex.Store({
         console.error(error);
       }
     },
+    async getMovieActors(_, movieId) {
```

Comment:
Why we use _ in argument?

Reply:
If I don`t use {commit} I should write "_," to avoid errors with axios.